WordPress/Classes/ViewRelated/Views/WPRichText/WPRichContentView.swift

```diff
@@ -178,7 +178,6 @@ private extension WPRichContentView {
             "em, i { font:-apple-system-body; font-family: 'Noto Serif'; font-weight: normal; font-style: italic; line-height:1.6; } " +
             "a { color: #\(style.linkColorHex); text-decoration: none; } " +
             "a:active { color: #\(style.linkColorActiveHex); } " +
-            "img.emoji { vertical-align: baseline; max-width: 1rem; } " +
```

Comment:
This was an experiment that I forgot to clean up. This line doesn't do anything to the `WPRichContentView`. 😅